redux_userlist_06/src/components/Dashboard/getList.tsx

```diff
@@ -0,0 +1,60 @@
+import "./styles.css";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { addUserList } from "../../actions/actions";
+import { UserDetails } from "../../interfaces";
```

Comment:
import sequence should be

react imports
any packages related imports
shared folders which are in the same app
utils and constants
styles
Fix all the imports in your project